common/src/main/java/org/jboss/da/common/version/VersionParser.java

```diff
@@ -48,7 +48,8 @@ public String removeRedhatSuffix(String version) {
 
     public Matcher getVersionMatcher(String version) {
         String nonRedhatVersion = removeRedhatSuffix(version);
-        return Pattern.compile(nonRedhatVersion + PATTERN_SUFFIX_BUILT_VERSION).matcher("");
+        return Pattern.compile(Pattern.quote(nonRedhatVersion) + PATTERN_SUFFIX_BUILT_VERSION)
+                .matcher("");
```

Comment:
@jbartece wraps string in regex quotes so we're safe from unsafe characters, as far as I understand